src/main/java/yevhenii/lostfilmdemo/quartz/LostfilmRssJob.java

```diff
@@ -0,0 +1,46 @@
+package yevhenii.lostfilmdemo.quartz;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.quartz.Job;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.springframework.beans.factory.annotation.Autowired;
+import yevhenii.lostfilmdemo.convertors.TVSeriesConvertor;
+import yevhenii.lostfilmdemo.services.FeedService;
+import yevhenii.lostfilmdemo.services.TVSeriesService;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+@Log4j2
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class LostfilmRssJob implements Job {
+
+    private final FeedService feedService;
+    private final TVSeriesConvertor convertService;
+    private final TVSeriesService seriesService;
+
+    @Override
+    public void execute(JobExecutionContext context) throws JobExecutionException {
+        log.info("on execute");
+        feedService.readFeed(getUrl())
+                .forEach(fs -> seriesService.save(convertService.convert(fs)));
+        log.info("end executing");
+    }
+
+    private String getUrl(){
+        String propertyFilePath = "config.properties";
```

Comment:
place your property inside application.properties and here just inject it using @Value annotation